test/snyk-to-html.test.ts

```diff
@@ -937,6 +937,30 @@ test('test snyk-to-html container app vulnerabilities', (t) => {
   );
 });
 
+test('test snyk-to-html consistent ignores suppressions', (t) => {
+  t.plan(2);
+  // report generated with "snyk code test --json" against snyk cli
+  // with issues suppressed
+  SnykToHtml.run(
+    path.join(__dirname, 'fixtures', 'test-code-consistent-ignores.json'),
+    noRemediation,
+    path.join(__dirname, '..', 'template', 'test-report.hbs'),
+    noSummary,
+    (report) => {
+      t.contains(
+        report,
+        '<div class="suppression-card">',
```

Comment:
**suggestion:** To better protect ourselves from introducing a regression we could add a guard assertion to one of our existing test cases to check that the output `doesNotHave`   <div class="suppression-card"> in the output.